app/assess/templates/components/projects_table/projects_table.html

```diff
@@ -0,0 +1,103 @@
+{% from "govuk_frontend_jinja/components/input/macro.html" import govukInput %}
+
+
+{% macro render(applications, q, application_search_endpoint) -%}
```

Comment:
so i think im right in thinking 'q' has been used here because we don't want repetition everywhere - but I'm all for explicitly naming variables

Reply:
I agree, this is a very ambiguous name. Let's keep it 🐍 y .

Reply:
Mutiny...! I will change it